components/splash/cards/whatIsFBCards.js

```diff
@@ -37,16 +37,17 @@ const Card = ({ icon, text, heading }) => (
     fontSize='0.875rem'
     lineHeight='1.3'
   >
+    <Icon name={icon} size='1.25rem' margin='0 0 .75rem' />
```

Comment:
why this change? i personally don't like it as much, what do you think?

Reply:
i don't like it either

Reply:
OK I'll revert that. Eryn noted it in her changes and it was in the original design, but I agree, I think it looks better how we had it.